app/assets/javascripts/application.js

```diff
@@ -14,3 +14,33 @@
 //= require jquery_ujs
 //= require turbolinks
 //= require_tree .
+$(document).ready(function() {
+  $('#add_comm').click(function(event){
+    event.preventDefault();
+
+    $(this).hide();
+    console.log(this)
+    var url = $(this).attr('href');
+
+    $.get(url, function(serverResponse, status, request){
+      console.log(serverResponse)
+      $('#comments').append(serverResponse);
+    });
+  });
```

Comment:
Think this is not used? In any case, in general, it's better ux to show / hide / dynamically create forms with JS rather than get+partial (it's a good idea to avoid a network round-trip if we can.)